cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -42,8 +42,13 @@ def get_clusters(appliance, provider, source_provider):
         target_cluster = provider.data.get("clusters")[0]
     except IndexError:
         pytest.skip("Cluster not found in given provider data")
+
+    clusters_obj = appliance.rest_api.collections.clusters.all
+    if provider.one_of(OpenStackProvider):
```

Comment:
Required: Please Add one `comment` explaining why we combining clusters and cloud tenants for OpenStack provider.